src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.ts

```diff
@@ -18,6 +19,12 @@ import { NotificationType } from '~/app/shared/enum/notification-type.enum';
 import { NotificationService } from '~/app/shared/services/notification.service';
 import { Router } from '@angular/router';
 import { map, switchMap } from 'rxjs/operators';
+import { SummaryService } from '~/app/shared/services/summary.service';
+import { ExecutingTask } from '~/app/shared/models/executing-task';
+import {
+  STEP_TITLES_MULTI_CLUSTER_CONFIGURED,
+  STEP_TITLES_SINGLE_CLUSTER
+} from '~/app/shared/enum/multisite-wizard-steps.enum';
```

Comment:
I don't get this. It goes against encapsulation. You have an abstract/generic Wizard component, then you have the specific RGWWizard component, right? But then you have some Enums in a generic directory (`shared/enum`) that are only valid for this component... What's the sense of it? The only place I recall where we're doing this was for the i18n button labels, and that was exactly to foster reuse of the same button labels...

Reply:
Moved the enum to rgw-multisite-wizard component.